src/Compilers/Core/Portable/Syntax/SyntaxAnnotation.cs

```diff
@@ -18,7 +18,7 @@ public sealed class SyntaxAnnotation : IObjectWritable, IObjectReadable, IEquata
         /// <summary>
         /// A predefined syntax annotation that indicates whether the syntax element has elastic trivia.
         /// </summary>
-        public static readonly SyntaxAnnotation ElasticAnnotation = new SyntaxAnnotation();
+        public static SyntaxAnnotation ElasticAnnotation { get; } = new SyntaxAnnotation();
```

Comment:
Actually, just changed my mind :) This might be a good candidate for lazy construction as @VSadov says. Leaving it as a property.